metricbeat/module/mongodb/_meta/config.yml

```diff
@@ -11,6 +11,21 @@
   # password configuration options.
   hosts: ["localhost:27017"]
 
+  # Optional SSL. By default is off.
+  #ssl.enabled: true
+
+  # Mode of verification of server certificate ('none' or 'full')
+  #ssl.verification_mode: 'full'
+
+  # List of root certificates for HTTPS server verifications
```

Comment:
Probably should say TLS instead of HTTPS (since there's no HTTP connection here)?

Reply:
You are right, fixed.